src/MagentoHackathon/Composer/Magento/Plugin.php

```diff
@@ -200,6 +214,9 @@ public function onNewCodeEvent(Event $event)
         $magentoModules = array_filter(
             $this->composer->getRepositoryManager()->getLocalRepository()->getPackages(),
             function (PackageInterface $package) use ($packageTypeToMatch) {
+                if ($package instanceof AliasPackage) {
+                    return false;
```

Comment:
No, it's in the callback for the `array_filter` each entry is passed to the callback, returning false will omit it from the resulting array. I am open to suggestions on how to make it more readable.